selenium_testing_library/screen.py

```diff
@@ -40,15 +42,23 @@ class NoSuchElementException(Exception):
 }
 
 
+class ElementsFinder(Protocol):
+    def find_elements(
+        self, by: str = locators.By.ID, value: str = None
+    ) -> List[WebElement]:
+        ...
+
+
 class Screen:
-    def __init__(self, driver: Driver):
+    def __init__(self, driver: RemoteWebDriver):
         self.driver = driver
+        self._finder: ElementsFinder = driver
```

Comment:
Feels a little bit dirty but what I actually want is the following:

driver can be any object that you pass in but for the screen functions to work it **needs** to have a `find_elements` method. Using `self.driver` should give you access to all the methods available on the object that you passed in, but for my internal functions I want to make sure that `find_elements` is definitely there.